source/tomopy/recon/wrappers.py

```diff
@@ -291,7 +291,7 @@ def ufo_dfi(tomo, center, recon, theta, **kwargs):
     Reconstruct object using UFO's Direct Fourier pipeline
     """
     import gi
-    gi.require_version('Ufo', '0.0')
+    # gi.require_version('Ufo', '0.0')
```

Comment:
I think version 0 is the only UFO version available. Maybe you meant to only remove the MINOR version?

```suggestion
    gi.require_version('Ufo', '0')
```

Any comments from @matze? This is legacy code that I have never touched.

Reply:
For me this actually creates an error `ValueError: Namespace Ufo not available for version 0`. As @matze suggested, could you elaborate on the error you get please?